### DISCUSSION TITLE
Colorscripts

### DISCUSSION THREAD

**Discussion by semi710** [marlonrichert/zsh-snap#264] [Q&A] [answered]
Correct way of adding colorscripts?

**Comment by marlonrichert** [comment]
No idea what you mean. Please edit your question to explain.

**Reply by semi710** [reply]
colorscripts are a collection of scripts those print some kind of art.
consider them like neofetch.
what i want to acheive is get neofetch output everytime i open terminal.
it seems znap resets promt and then it get's erased if i put the command after prompt init.

**Reply by marlonrichert** [reply, marked as answer]
Have you tried running the script _before_ running `znap prompt`?

**Reply by semi710** [reply]
i tried it every where just not in between these two lines where it worked.
```zsh
znap eval starship 'starship init zsh --print-full-init'
znap prompt
```
Thanks mate
